Sources/EpoxyCore/Model/Providers/ErasedContentProviding.swift

```diff
@@ -32,18 +32,18 @@ extension EpoxyModeled where Self: ErasedContentProviding {
   /// A closure that can be called to determine whether the given `model`'s `erasedContent` is equal
   /// to this model's `erasedContent`, else `nil` if there is no content or the content is always
   /// equal.
-  public var isErasedContentEqual: ((Self) -> Bool)? {
+  public var isErasedContentEqual: ((ErasedContentProviding) -> Bool)? {
     get { self[isContentEqualProperty] }
     set { self[isContentEqualProperty] = newValue }
   }
 
   // MARK: Private
 
   private var contentProperty: EpoxyModelProperty<Any?> {
-    .init(keyPath: \Self.erasedContent, defaultValue: nil, updateStrategy: .replace)
+    .init(keyPath: \ErasedContentProviding.erasedContent, defaultValue: nil, updateStrategy: .replace)
```

Comment:
I wondered if changing the keys could break existing code. I see that [`EpoxyModelStorage`](https://github.com/airbnb/epoxy-ios/blob/e46fab4024d4e8e0d5754d7add8be1ccb94390f3/Sources/EpoxyCore/Model/EpoxyModelStorage.swift) does not provide a way to enumerate keys, which increases my confidence that this change is safe.